src/main/java/mil/dds/anet/database/mappers/AttachmentMapper.java

```diff
@@ -0,0 +1,23 @@
+package mil.dds.anet.database.mappers;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import mil.dds.anet.beans.Attachment;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public class AttachmentMapper implements RowMapper<Attachment> {
+  @Override
+  public Attachment map(ResultSet rs, StatementContext ctx) throws SQLException {
+    final Attachment a = new Attachment();
+    MapperUtils.setCommonBeanFields(a, rs, null);
+    a.setMimeType(rs.getString("mimeType"));
+    a.setAuthorUuid(rs.getString("authorUuid"));
+    a.setContent(rs.getBytes("content"));
```

Comment:
We probably should never do this, as it will (try to) retrieve the entire file contents into memory. Better to use `final Blob contentBlob = rs.getBlob("content")` (maybe just in the few places where we want the contents, and not in the general mapper), and use `contentBlob.getBinaryStream()` to stream the contents directly into the HTTP response.

Reply:
Similary we should perhaps split sending the attachment metadata (with GraphQL) and uploading the file contents (through a plain API call) so we can use `setBinaryStream()` on the insert statement.